.gitmodules

```diff
@@ -20,4 +20,4 @@
 [submodule "windows_docker_resources/ros2-cookbooks"]
 	path = windows_docker_resources/ros2-cookbooks
 	url = git@github.com:ros-infrastructure/ros2-cookbooks
-	branch = latest
+	branch = clalancette/pin-pydot
```

Comment:
This branch and the submodule commit need to be updated once https://github.com/ros-infrastructure/ros2-cookbooks/pull/58 is merged.